perseo-main/src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -24,6 +24,8 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Properties;
+import java.util.ResourceBundle;
```

Comment:
It seem that after last changes ResourceBunle is no longer used in this file. Thus, this import line should be removed.

Reply:
Fixed in b6d7148bbc37f5c8a5de01bf50c6c12a6d80d237